iam/kc_config.json

```diff
@@ -26,7 +26,7 @@
   "oauth2DeviceCodeLifespan" : 600,
   "oauth2DevicePollingInterval" : 5,
   "enabled" : true,
-  "sslRequired" : "external",
+  "sslRequired" : "NONE",
```

Comment:
Isn't it risky to set it to None?

Reply:
no it is not in this case. It's for localhost cytomine urls